core/domain/user_services.py

```diff
@@ -775,19 +775,72 @@ def get_user_contribution_rights(user_id):
         UserContributionRights. The UserContributionRights domain object for the
         corresponding user.
     """
-    user_model = (
-        user_models.UserContributionRightsModel.get_by_id(user_id))
-    if user_model is not None:
-        return user_domain.UserContributionRights(
-            user_id,
-            user_model.can_review_translation_for_language_codes,
-            user_model.can_review_voiceover_for_language_codes,
-            user_model.can_review_questions)
-    else:
-        return user_domain.UserContributionRights(user_id, [], [], False)
+    return get_users_contribution_rights([user_id])[0]
+
+
+def get_users_contribution_rights(user_ids):
```

Comment:
Since this is a new public API function, it should have tests. Otherwise, suggest making it private if you don't actually want or need it to be part of the service's public API.

Reply:
In general it's fine, but from a purist perspective of "each API method should have its own isolated test suite" this doesn't quite fit, which was part of my concern. Introducing more thorough tests later isn't ideal, so you may want to consider making this private in this PR and then exposing it as a public API method with dedicated tests later.